src/test/resources/30/single/opf/invalid/prefixes-001.opf

```diff
@@ -8,8 +8,7 @@
         <dc:language>la</dc:language>
         <dc:date>2011-09-01</dc:date>
         <meta property="dcterms:modified">2011-09-01T17:18:00Z</meta>
-		<meta property="meta-auth">Auth</meta>
-		<meta property="media:test meta-auth">test</meta>
```

Comment:
this test case was used to verify that multiple property values were allowed and detected; we should replace `meta-auth` by another valid property value.

Reply:
Is there documentation of what these tests are for? I assumed by being an "invalid" test with the name "prefixes", it was checking for the undeclared "media:" prefix.

Reply:
Isn't this the only relevant error in the file:

Reply:
But going to the source again, isn't this the duplication error being checked:

```
<link rel="marc21xml-record foaf:ddd foaf:ddd" href="pub/meta/nor-wood-marc21.xml"/>
```

Reply:
> Is there documentation of what these tests are for?

No. We have to "guess" from the test file’s or test method’s names or contents. Which is another reasons why test maintenance is difficult 🤷‍♂️

> I assumed by being an "invalid" test with the name "prefixes", it was checking for an undeclared prefix.

Possibly. I'll go over all these and try to figure this out!